interfaces/cython/cantera/onedim.pyx

```diff
@@ -837,37 +837,51 @@ cdef class Sim1D:
         nPoints = [len(flow_domains[0].grid), 12, 24, 48]
 
         for N in nPoints:
+            if N > self.get_max_grid_points():
```

Comment:
On this line, I cannot figure out why `self.get_max_grid_points()` doesn't require an argument for the domain. Putting in an additional argument raises a `TypeError` that there are too many arguments. If you have any ideas, I'd love to hear them.

That said, this works as expected, so I just left it this way...